docs/table_design/dynamic_partitioning.md

```diff
@@ -35,16 +35,19 @@ PROPERTIES(
+|-----------------------------------------| -------- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dynamic_partition.enable                | No       | Enables dynamic partitioning. Valid values are `TRUE` and `FALSE`. The default value is `TRUE`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| dynamic_partition.time_unit             | Yes      | The time granularity for dynamically created  partitions. It is a required parameter. Valid values are `HOUR`, `DAY`, `WEEK`, `MONTH`, and `YEAR`.The time granularity determines the suffix format for dynamically created partitions.<br/>  - If the value is `DAY`,  the suffix format for dynamically created partitions is yyyyMMdd. An example partition name suffix is `20200321`.<br/>  - If the value is `WEEK`, the suffix format for dynamically created partitions is yyyy_ww, for example `2020_13` for the 13th week of 2020.<br/>  - If the value is `MONTH`, the suffix format for dynamically created partitions is yyyyMM, for example `202003`.<br/> - If the value is `YEAR`, the suffix format for dynamically created partitions is yyyy, for example `2020`. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/e1785644-df9f-4c55-b263-997762301cb6)
should be <ul> and <li> tags